README.md

```diff
@@ -122,6 +122,13 @@ Please refer to the demos of the MQTT client library in the following locations
 
 ## Generating documentation
 
+Note: For pre-generated documentation, please see the documentation linked in the above repositories:
```

Comment:
It may be better to have the table under a different section than "Generating Documentation" as Abhijit had mentioned that the main concern is that it makes it misleading for customers about them having to generate documentation. 

If the table is present in a separate section, like "Existing Documentation", toward the beginning of the README, it would be easier for customers to notice

Reply:
How about this?

# Documentation

## Existing Documentation

## Generating Documentation

Reply:
Yes, that's fine. These can be moved before the Unit Tests section though to make it more prominent for customers to find